src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -231,11 +232,27 @@ const Occurrences: React.FC<Props> = ({
   ];
 
   const renderOccurrenceInfo = (occurrence: OccurrenceFieldsFragment) => {
-    const { placeId, startTime, endTime } = occurrence;
+    const { placeId, startTime, endTime, linkedEvent } = occurrence;
     const date = formatDate(new Date(startTime));
     const time =
       startTime &&
       formatTimeRange(new Date(startTime), new Date(endTime), locale);
+    const offer = linkedEvent?.offers?.[0];
+    const price = offer?.price?.[locale] || '';
+    const priceDescription = offer?.description?.[locale] || '';
+    const isFree = offer?.isFree ?? !price;
+    const priceInfoUrl = offer?.infoUrl?.[locale] || '';
```

Comment:
all those default values after seems to be little bit redundant, I would suggest something like this:

```
    const offer = linkedEvent?.offers?.[0];
    const price = offer?.price?.[locale];
    const priceDescription = offer?.description?.[locale];
    const isFree = offer?.isFree ?? !price;
    const priceInfoUrl = offer?.infoUrl?.[locale];
```

Reply:
Changed.